bitbots_vision/config/visionparams.yaml

```diff
@@ -3,6 +3,7 @@
 vision_parallelize: true  # Run the neural net and the conventional part in parallel
 vision_blind_threshold: 30  # Brightness threshold under which the vision thinks, that someone forgot the camera cap
 
+ROS_audio_msg_topic: 'speak'  # ROS topic of the audio message
```

Comment:
Why is speak added here?

Reply:
> Why is speak added here?

I don't know but that was merged from the current master